nncf/internal/deprecation.py

```diff
@@ -0,0 +1,42 @@
+""""
```

Comment:
Hello, new `utils` package :) What do you think about moving deprecation.py to nncf.common.utils?

Reply:
This is internal code. Users are not supposed to touch this. I can make this even more obvious by calling the top-level module `_internal`, but then we will have problems with pylint, unless we ignore all `import-private-name` warnings.

Reply:
First of all, we should think about is the friendly and intuitive modules structure in NNCF (https://github.com/openvinotoolkit/nncf/tree/develop/nncf), and if there is an opportunity not to add a new module, then It's better to do so.